terraform/aws/apps-deployment/main.tf

```diff
@@ -1,10 +1,10 @@
 terraform {
-  required_version = "0.14.5"
+  required_version = ">= 1.0.0"
```

Comment:
0/5 should the version be fixated? I suppose the terraform version rarely needs to be updated. Fixating the version might help avoid unexpected changes in terraform.

Reply:
@hanzei It is safe and good way to do it because we control the Terraform version in the Makefile, so its should not break.